src/build.tsx

```diff
@@ -146,7 +146,8 @@ const perfLoadStart = performance.now() // For telemetry.
 
 			if (isProduction) {
 				const customDimensions = {
-					logLength: logs.length + '',
+					results: logs.reduce((accum, log) => accum + log.runs.reduce((accum, run) => accum + run.results?.length ?? 0, 0), 0).toString(),
+					logs: logs.length.toString(),
```

Comment:
is there a chance any existing telemetry queries depend on "logLength"? rather than change to "logs" perhaps keeping as "logLength"?

i do prefer "logs", but just a thought 👍

Reply:
We're starting telemetry from scratch so there's nothing existing!